app/components/ConnectDapp/ApproveTransaction.js

```diff
@@ -427,6 +427,8 @@ const ApproveTransaction = ({
                   String(request.params.request.params.signers[0]?.scopes)
                 ] === 'Global' && <WarningIcon />}
               </div>
+            ) : (
+              <div>{WITNESS_SCOPE['1']}</div>
```

Comment:
Shouldn't this be `WITNESS_SCOPE[1]`? Why is this a string

Reply:
Because the key for this const is a string. if you look on the line 28, the WITNESS_SCOPE is a const.